conjure-java-core/src/test/java/com/palantir/conjure/java/JerseyServiceEteTest.java

```diff
@@ -63,12 +63,11 @@
 import org.junit.jupiter.api.io.TempDir;
 import org.junit.jupiter.api.parallel.Execution;
 import org.junit.jupiter.api.parallel.ExecutionMode;
-import org.junit.jupiter.api.parallel.ResourceLock;
 
 @Execution(ExecutionMode.CONCURRENT)
 @ExtendWith(DropwizardExtensionsSupport.class)
-@ResourceLock("port:8080")
```

Comment:
I'd have kept this since we do need to guarantee nothing else uses port 8080, but in practice there are no other uses of `@ResourceLock` in this project.

Reply:
Fair enough